src/less/components/payex/togglebox.less

```diff
@@ -0,0 +1,44 @@
+@import "../togglebox.less";
+
+/* For theme-white */
+.togglebox:has(input.theme-white) input[type="checkbox"]:not(:disabled) + label:after {
+    background-color: var(--brand-secondary);
+}
+
+.togglebox:has(input.theme-white) input[type="checkbox"]:not(:disabled) + label:before {
+    background-color: var(--white);
+}
+
+.togglebox:has(input.theme-white) input[type="checkbox"]:not(:disabled):checked + label:before {
+    background-color: var(--brand-success);
+}
+
+.togglebox:has(input.theme-white) input[type="checkbox"]:not(:disabled):checked + label:after {
+    background-color: var(--white);
+}
+
+/* Theme-white background */
+.documentation 
+.showcase-panel.showcase-panel-advanced 
+.component-preview 
+.component-preview-content:has(input[type="checkbox"].theme-white) {
+    background-color: var(--bg-togglebox-theme-white) !important;
```

Comment:
Denna !important går inte att komma ifrån va? 😜

Reply:
if this is 1 single selector, with no comma, then keep it all on the same line

Reply:
AND if this rule is about the display of the documentation preview then it should not be specified here (since those rules will be shipped in the npm package) but instead be declared in the LESS file used for the documentation website only (which rules will not be used for the npm package)

-> "src\less\documentation-payex.less"